app/src/main/java/org/openobservatory/ooniprobe/test/TestAsyncTask.java

```diff
@@ -149,10 +149,19 @@ private void downloadURLs() {
             OONISession session = EngineProvider.get().newSession(EngineProvider.get().getDefaultSessionConfig(
                     app, BuildConfig.SOFTWARE_NAME, BuildConfig.VERSION_NAME, new LoggerArray(), proxy));
             OONIContext ooniContext = session.newContextWithTimeout(30);
+
+            ThirdPartyServices.addLogExtra("ooniContext", app.getGson().toJson(ooniContext));
+
             session.maybeUpdateResources(ooniContext);
             OONIURLListConfig config = new OONIURLListConfig();
             config.setCategories(app.getPreferenceManager().getEnabledCategoryArr().toArray(new String[0]));
+
+            ThirdPartyServices.addLogExtra("config", app.getGson().toJson(config));
+
             OONIURLListResult results = session.fetchURLList(ooniContext, config);
+
+            ThirdPartyServices.addLogExtra("results", app.getGson().toJson(results));
```

Comment:
Is it correct to assume that this will be sent to sentry only in the event of an exception, but in all cases it will not?